awscli/customizations/cloudformation/deploy.py

```diff
@@ -276,7 +286,8 @@ def _run_main(self, parsed_args, parsed_globals):
                 "s3",
                 config=Config(signature_version='s3v4'),
                 region_name=parsed_globals.region,
-                verify=parsed_globals.verify_ssl)
+                verify=parsed_globals.verify_ssl
```

Comment:
@baronswindle Looks like we're missing a comma at the end of this line. Seems to be breaking the builds currently:
```======================================================================
ERROR: Failure: SyntaxError (invalid syntax (deploy.py, line 290))
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/travis/build/aws/aws-cli/tests/unit/customizations/cloudformation/test_deploy.py", line 20, in <module>
    from awscli.customizations.cloudformation.deploy import DeployCommand
  File "/home/travis/build/aws/aws-cli/awscli/customizations/cloudformation/__init__.py", line 14, in <module>
    from awscli.customizations.cloudformation.deploy import DeployCommand
  File "/home/travis/build/aws/aws-cli/awscli/customizations/cloudformation/deploy.py", line 290
    endpoint_url=parsed_args.s3_endpoint_url)
               ^
SyntaxError: invalid syntax
```

Reply:
@whummer My apologies - that was careless. Just pushed up the fix.